src/controllers/expense.controller.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
+
+const expenseService = require('../services/expense.service');
+
+const get = (req, res) => {
+  const { userId, categories, from, to } = req.params;
+
+  res
+    .status(200)
+    .send(expenseService.get(userId, categories, from, to));
+};
+
+const post = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  const result = expenseService.post(
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note || '',
+  );
+
+  switch (result) {
+    case 'No userId': {
+      return res
+        .send('No userId')
+        .status(400);
+    }
+
+    case 'No spentAt': {
+      return res
+        .send('No spentAt')
+        .status(400);
+    }
+
+    case 'No title': {
+      return res
+        .send('No title')
+        .status(400);
+    }
+
+    case 'No amount': {
+      return res
+        .send('No amount')
+        .status(400);
+    }
+
+    case 'No category': {
+      return res
+        .send('No category')
+        .status(400);
+    }
+
+    case 'No expense to update': {
+      return res
+        .send('No expense to update')
+        .status(400);
+    }
```

Comment:
would be better to check if all params provided in the controller and call the service method only if everything is ok